text/0018-export_behavior.md

```diff
@@ -0,0 +1,145 @@
+
+# Motivation
+[motivation]: #motivation
+
+An <a href="https://github.com/qri-io/rfcs/blob/master/text/0014-export.md">earlier RFC</a> began describing the export functionality. However, it left a number of details unspecified, which led to too much uncertainty to move forward with implementation. While it did mention many of the options for export, it left out discussions of defaults, the interactions between options, and basically how `export` is expected to work. This RFC hopes to remedy this situation so that implementation can proceed without hesitation.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Export is a piece of functionality that will generate the version of a dataset which is external to qri storage, so that it can live instead on a local filesystem. The goal is to have a document defintion of the structured data that has been, up until now, represented only internally by qri. Here are some of the use cases that `export` is meant to handle:
+
+* Getting a local binary copy of a dataset, such that it can be safetly deleted from qri, and later imported without any loss of fidelity or history.
+* Exporting an entire dataset with metadata and body to analyze in a custom script. A single file "json" or "yaml", for example.
+* Converting to a binary format (like xlsx) for use in another program like Excel or OpenOffice.
+* Deciding to stop using qri, and wanting to get all of your data out in a high-fidelity form, with both machine and human readable formats.
+
+These should all be doable using `export`.
+
+Some non-goals:
+
+* Retrieving only a single component of a dataset, like the title.
+* Converting a dataset's body to a different format than it is being stored as (csv to json or cbor).
+
+These types of operations should be added to `get` instead, which already has similar functionality.
+
+## Formats
+
+Each of the above listed use cases for `export` corresponds to a different export format. Here they are discussed in more detail.
+
+### Native export
+
+The default option is a native export. This consists of a zip file containing two things, a `manifest.cbor` file, then a list of binary blocks containing the data that is directly read from IPFS. The manifest simply enumerates these blocks. This native format is designed to work best with reimports; using it will recreate the dataset with no loss of information.
+
+### Foreign export
+
+An `export` can also be foreign, using the `--format` flag to choose a specific file format. For example:
+
+```
+qri export --format json me/my_dataset
+```
+
+```
+qri export --format xlsx me/my_dataset
+```
+
+A file created by these commands still contains as much data about the original dataset, but reimporting it is not guaranteed to end up with the exact same result as the dataset it was exported from.
+
+For the case of "json", the result will be a single file, with the body stored inline in the document.
```

Comment:
Can we get a quick addition here clarifying that the only way to re-import a foreign export is to create a new version of a dataset using `qri save`?